components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const externalLinks = [
+    {
+      href: 'https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf',
+      translationId: 'trademarkPolicy',
+    },
+    {
+      href: 'https://openjsf.org/wp-content/uploads/sites/84/2021/04/OpenJS-Foundation-Privacy-Policy-2019-11-15.pdf',
+      translationId: 'privacyPolicy',
+    },
+    {
+      href: 'https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md',
+      translationId: 'codeOfConduct',
+    },
+    {
+      href: 'https://github.com/nodejs/node/blob/HEAD/SECURITY.md#security',
+      translationId: 'security',
+    },
+  ];
+
+  const socialLinks = [
+    {
+      href: 'https://github.com/nodejs/node',
```

Comment:
Same here with social links but the object entry would be `{ icon: string, link: string, kind: 'dark' | 'light' | 'neutral', alt?: string }`

Reply:
@ovflowd Do you have any suggestions to solve this TS error when typing the `kind` property from the JSON file?
![image](https://github.com/nodejs/nodejs.org/assets/19715103/42c8d644-1698-477b-8720-b97bb83f1c34)

Reply:
Is this still happening? Where exactly does this error happen?

Reply:
this error is shown in `siteProvider.tsx`. I guess we could use typecasting to remove the error, but then there would be no type safety